lib/Twig/Extensions/Extension/Text.php

```diff
@@ -55,24 +55,43 @@ function twig_truncate_filter(Twig_Environment $env, $value, $length = 30, $pres
 
     function twig_wordwrap_filter(Twig_Environment $env, $value, $length = 80, $separator = "\n", $preserve = false)
     {
-        $sentences = array();
-
         $previous = mb_regex_encoding();
         mb_regex_encoding($env->getCharset());
 
-        $pieces = mb_split($separator, $value);
-        mb_regex_encoding($previous);
+        $lines = mb_split($separator, $value);
```

Comment:
renaming ``$pieces`` to ``$lines`` is bad, because nothing forces the separator to be a newline char. Please revert this renaming.